src/components/Body.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState, useEffect } from "react";
+import MovieCard from "./MovieCard";
+import axios from "axios";
+import "../styles/Body.css";
+import deathStar from '../assets/death-star.png'
+
+function Body() {
+  const [data, setData] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  // useEffect(() => {
+  //   fetch(`https://swapi.dev/api/films`)
+  //     .then((response) => {
+  //       // if (!response.ok) {
+  //       //   throw new Error(`Page not found: Error ${response.status}`);
+  //       // }
+  //       // console.log(response.json() )
+  //       console.log(response)
+  //       return response.json();
+  //     })
+  //     .then((actualData) => {
+  //       setData(actualData);
+  //       // setError(null);
+  //     })
+  //     .catch((error) => {
+  //       setError(error.message);
+  //       setData(null);
+  //     })
+  //     .finally(() => {
+  //       setLoading(false);
+  //     });
+  // }, []);
+
+  useEffect(() => {
+    const getData = async () => {
+      try {
+        const response = await axios.get(`https://swapi.dev/api/films`);
+        setData(response.data);
+        // setError(null);
```

Comment:
Any reason for commenting this line?

Reply:
That was when I was testing out the code, it seemed it wasn't doing anything even when commented out but I later figured out it's function.